tests/e2e/helpers/conf.ts

```diff
@@ -19,6 +19,14 @@ export const ossStandaloneConfig = {
     databasePassword: process.env.OSS_STANDALONE_PASSWORD
 };
 
+export const ossStandaloneConfig2 = {
```

Comment:
It would be better to name it as ossStandaloneConfigEmpty or something like that